portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-download.controller.js

```diff
@@ -50,7 +56,7 @@ const getReportDownload = async (req, res) => {
 
     return readStream.pipe(res);
   } catch (error) {
-    const errorMessage = `Failed to get utilisation report for download with _id '${_id}'`;
+    const errorMessage = `Failed to get utilisation report for download with id '${id}'`;
     console.error(errorMessage, error);
     return res.status(error.response?.status ?? 500).send({ message: errorMessage });
```

Comment:
Out of interest, do we want to use `.json` here? It might be worth seeing if any of the sca we have picks up on this as a risk

Reply:
Fwiw I'm used to just using send for everything as it equates to the same thing, it just makes the code more explicit to do .json but it end result is the same I think. Not sure why it would be a risk has it been flagged before?